# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaDecoderOwner.h
# Commit: 243a33803d16
# Full Hash: 243a33803d16d50cfcc8cf0a5c4739a5ed08dc55
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2018-11-23 21:37:23
# Regressor Bug: 1172394
# File Overlap Count: 1
# Description:
#   Bug 1423241 - Refactor DecodedStream. r=jya
#   
#   This removes DecodedStream's use of MediaStreamListener in favor of
#   MediaStreamTrackListener. This change has however rippled through to a lot
#   more cleanup, per below.
# ==============================================================================

diff -r 12f91d8a1f69 -r 243a33803d16 dom/media/MediaDecoderOwner.h
--- a/dom/media/MediaDecoderOwner.h	Fri Nov 23 15:01:48 2018 +0000
+++ b/dom/media/MediaDecoderOwner.h	Fri Nov 23 15:02:03 2018 +0000
@@ -10,6 +10,7 @@
 #include "MediaInfo.h"
 #include "MediaSegment.h"
 #include "nsSize.h"
+#include "TrackID.h"
 
 class nsIDocument;
 